app/models/user.rb

```diff
@@ -1121,7 +1117,7 @@ def url
     only: ["id", "wca_id", "name", "gender",
            "country_iso2", "created_at", "updated_at"],
     methods: ["url", "country", "delegate_status"],
-    include: ["avatar", "teams"],
+    include: ["avatar"],
```

Comment:
This PR would be a good opportunity to think about optimizing the `deprecated_team_roles` method.

As a matter of fact, currently there is a snippet of code in the serialization logic for users that reads
```ruby
if include_teams
  json[:teams] = deprecated_team_roles
end
```

and this will become silent (or "dead") code after this change. Some parts of our application may break if they rely on team roles being serialized to the user json.

Reply:
(as a matter of fact, that's why some of the non-choke CI tests fail)

Reply:
Oh ya I forgot that this was anyway loading made-up teams and not the actual teams. My bad. Added it back.

Regarding optimizing `deprecated_team_roles`, there is already some optimization in https://github.com/thewca/worldcubeassociation.org/pull/9411, but I don't remember how many queries it's taking now. Once that PR is merged, I'll take up that as a different PR.